disco_aws_automation/disco_elastigroup.py

```diff
@@ -0,0 +1,291 @@
+"""Contains DiscoElastigroup class that orchestrates AWS Spotinst Elastigroups"""
+import logging
+import time
+import json
+from os.path import expanduser
+from base64 import b64encode
+import requests
+
+from .exceptions import TooManyAutoscalingGroups
+
+logger = logging.getLogger(__name__)
+
+SPOTINST_API = 'https://api.spotinst.io/aws/ec2/group/'
+
+
+class DiscoElastigroup(object):
+    """Class orchestrating elastigroups"""
+
+    def __init__(self, environment_name, token=None, session=None):
+        self.environment_name = environment_name
+        self._token = token or None
+        self._session = session or None
+        self.account_id = '646102706174'
```

Comment:
We should pull this from `disco_aws.ini` we can add a new config if it doesn't exist